python/lsst/pipe/tasks/diff_matched_tract_catalog.py

```diff
@@ -37,6 +37,7 @@
 
 from abc import ABCMeta, abstractmethod
 from astropy.stats import mad_std
+import astropy.table as apTab
```

Comment:
This doesn't seem like standard practice. It looks like you only need `Table` and `vstack` anyway, so I'd recommend changing to:
```
from astropy.table import Table, vstack
```
and modifying the relevant code that calls it below.

Reply:
This is a moot point now but I really prefer to have a comprehensible package prefix when importing a name as generic as `Table` (though fair enough if you think `apTab` is too concise, but it is in line with how we shorten stack package names).

Reply:
Nevertheless, `from astropy.table import Table` is a fairly standard practice across the stack, and we do seem to be moving away from `import X as Y` imports. It also helps a) keep line lengths shorter, which b) improves readability (imo), and c) makes it clear if an extra name is now required on a PR following a subsequent edit.